frontend/scripts/generateDocs.ts

```diff
@@ -1,24 +1,41 @@
 import fs from 'fs'
 import path from 'path'
+import { execSync } from 'child_process'
+
+const PACKAGE_NAME = '@acaldas/document-model-libs'
+
+try {
+  execSync(`npm ls ${PACKAGE_NAME}`)
+} catch (e) {
+  throw new Error(`Failed to generate documentation from the dependency (${PACKAGE_NAME}), did you run \`npm install\`?`)
+}
+
+// eslint-disable-next-line import/first
 import { concatMdSync } from 'concat-md'
 
-const DOCS_DIR = process.env.DOCS_DIR
+const DOCS_DIR = path.resolve(__dirname, '..', 'node_modules', ...PACKAGE_NAME.split('/'), 'dist', 'docs')
```

Comment:
Nit: you actually don't need to split paths into chunks, you only need to 1) avoid joining strings via `+` and 2) wrap them into some `path` method, which will take care of setting right delimiter per platform, [see example in the docs](https://nodejs.org/api/path.html#pathjoinpaths)
```suggestion
const DOCS_DIR = path.resolve(__dirname, '../node_modules', PACKAGE_NAME, 'dist/docs')
```